grammars/puppet.cson

```diff
@@ -146,7 +146,7 @@
     'include': '#variable'
   }
   {
-    'begin': '(?i)\\b(import|include|contain)\\b\\s*'
+    'begin': '(?!.*=>)\\b(import|include|contain|require)\\s+'
```

Comment:
I believe negative lookaheads need to be at the start of the regex.

The require function has to be all lowercase, uppercase strings are reserved as Meta Classes or Functions in Puppet, so REQUIRE, INCLUDE etc would be invalid syntax